test/awsE2ESolutionSetup.sh

```diff
@@ -133,25 +189,24 @@ if [ "$BOOTSTRAP_REGION" = true ] ; then
   bootstrap_region
 fi
 # Deploy source cluster on EC2 instances
-cdk deploy "*" --require-approval never --c suffix="Migration-Source" --c distVersion="7.10.2" --c distributionUrl="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.10.2-linux-x86_64.tar.gz" --c captureProxyEnabled=true --c captureProxyTarUrl="https://github.com/opensearch-project/opensearch-migrations/releases/download/1.0.0/trafficCaptureProxyServer.x86_64.tar" --c securityDisabled=true --c minDistribution=false --c cpuArch="x64" --c isInternal=true --c singleNodeCluster=true --c networkAvailabilityZones=2 --c dataNodeCount=2 --c managerNodeCount=0 --c serverAccessType="ipv4" --c restrictServerAccessTo="0.0.0.0/0"
```

Comment:
I'm guessing this flag was removed due to updating the `startCaptureProxy.sh` script and adding the ability to build the Capture Proxy on the node?

Reply:
That's right. This option could still be provided and it will download from our repo distributions, but when it is not we try and build the Proxy from source on the node